src/circuit/gadget/ecc/chip.rs

```diff
@@ -491,7 +495,11 @@ impl EccInstructions<pallas::Affine> for EccChip {
     fn mul_fixed_short(
         &self,
         layouter: &mut impl Layouter<pallas::Base>,
-        magnitude_sign: (CellValue<pallas::Base>, CellValue<pallas::Base>),
+        // TODO: Return AssignedCell<Sign, pallas::Base>
+        magnitude_sign: (
+            AssignedCell<pallas::Base, pallas::Base>,
+            AssignedCell<pallas::Base, pallas::Base>,
+        ),
```

Comment:
I suggest making `MagnitudeSign` a typedef for
`(AssignedCell<pallas::Base, pallas::Base>, AssignedCell<pallas::Base, pallas::Base>)` now, then changing it to `(AssignedCell<u64, pallas::Base>, AssignedCell<Sign, pallas::Base>)` later.

```suggestion
        magnitude_sign: MagnitudeSign,
```